utils/installation.sh

```diff
@@ -1,54 +1,85 @@
 #!/bin/bash
 
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 
+   exit 1
+fi
+
+apt update -y
```

Comment:
what if we have a different package manager?
we must differentiate among OSes

```suggestion
if [[ $OS_RELEASE == "ubuntu" ]] ; then
apt update -y
....
```